### PR TITLE
Split HSBC to add HSBC US

### PR DESCRIPTION
**Pull request by obscurans**

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -521,14 +521,22 @@ websites:
...
       - proprietary
       - hardware
     doc: https://www.hsbc.co.uk/help/security-centre/secure-key/secure-key-help/
 
+  - name: HSBC US
+    url: https://www.us.hsbc.com/
+    img: hsbc.png
+    twitter: HSBC_US
+    facebook: HSBCUS
+    status: https://www.us.hsbc.com/online-banking/security-device/
```
Comment (Carlgo11) [inline _data/banking.yml]: The status tag is designed to link users to a blog post or tweet from the company explaining when 2FA will be rolled out.
```suggestion
```

File: _data/banking.yml [hunk 2]
```diff
@@ -521,14 +521,22 @@ websites:
...
+    exception: "Proprietary device password OR normal account password logs in. Only special actions require both."
```
Comment (Carlgo11) [inline _data/banking.yml]: As their current system doesn't meet our [definition](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#a-note-on-definitions) of 2FA there's no reason to have the exception tag in the listing.
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/banking.yml (hunk 1 above)
Comment (obscurans) [reply _data/banking.yml]: Do we have a field for this specific purpose, ie explaining why a company that seemingly implemented 2FA didn't? (And still shame them)

**Comment by obscurans** [comment]
Do we have a field for this specific purpose, ie explaining why a company that seemingly implemented 2FA didn't? (And still shame them)

**Comment by phallobst** [comment]
Is the 2FA support by HSBC US really different from the other branches? (see #4253)

**Comment by obscurans** [comment]
I can't speak to HSBC non-US, but at least on Android they specifically split off their app from the rest of HSBC. As #3872 noted, this site doesn't actually have 2FA

**Pull request closed without merging**